package.json

```
@@ -12,7 +12,7 @@
     "version": "8.1.0"
```

```suggestion
    "version": "8.1.1"
```

This property should be bumped together with `devDependencies`.